cartoframes/data/observatory/catalog/dataset.py

```diff
@@ -497,3 +498,6 @@ def _get_summary_data(self):
         else:
             log.info('Summary information is not available')
             return None
+
+    def __str__(self):
+        return "<Dataset.get('{}')>".format(self._get_print_id())
```

Comment:
Why overriding this?

Reply:
It is a @arredond request. When an error happens, we are showing a very huge error message due to the big string created for the Dataset and basically it is difficult to understand